### ISSUE TITLE
Failed to integrate RGBD image into a TSDFVolume

### ISSUE THREAD

**Issue by HilbertXu** [isl-org/Open3D#4419]
Hi all,

I am try to integrating RGBD image to a TSDFVolume in Pybullet, for now, I have successfully generated point clouds from RGBD image with camera intrinsic matrix, the point cloud looks like this:
![Screenshot from 2021-12-13 15-26-44](https://user-images.githubusercontent.com/33152255/145852676-2548da35-b3e9-42a3-b337-b2433596d928.png)

For my setup, the camera is put just above the object sets, so the extrinsic matrix of camera looks like this:
![Screenshot from 2021-12-13 15-48-09](https://user-images.githubusercontent.com/33152255/145853062-3c7f9f84-1ebb-4e76-b7f4-5f18310996cc.png)

However, when I tried to integrate RGBD into a TSDFVolume, I found that the TSDFVolume after integrating contains 0 points, and no error occured. Do you have any suggestion about how to solve this problem? is this related to the size of TSDFVolume?

Thanks a lot!

**Comment by theNded** [comment]
Please first verify the depth scale and the max depth truncation. They vary from dataset to dataset. 
The default scale is 1000, meaning that we assume per pixel depth value is in millimeter. You may want to change that accordingly.

**Comment by HilbertXu** [comment]
Hi, 

For my setup, the depth is represented in meters, the max depth is 0.4450863, min depth is 0.3285103, and mean depth is0.42690668.

For the TSDF, I intialize a TSDF with size=2 and resolution=120, so the voxel size should be 2/120, and I set the sdf_trunc = 10 * voxel_size, are these parameters ok? 

In order to move the point clouds within the volume of TSDF, I transform the points cloud to the TSDF system which originate at [size/2, size/2, 0], is this correct? After that, I checked the transformed point clouds and seems like all points are within the volume of TSDF. But still, there is no point in the TSDF.

Thanks a lot for your help!
